model/metrics/http.yaml

```diff
@@ -124,3 +124,147 @@ groups:
+        requirement_level: recommended
+        brief: HTTP protocol version of the connection in the connection pool.
+        note: >
+          HTTP 1.0 and 1.1 requests share connections in the connection pool and are both reported as version `1.1`.
+          So, the `network.protocol.version` value reported on connection metrics is different than the one reported
+          on request-level metrics or spans for HTTP 1.0 requests.
+        examples: ["1.1", "2", "3"]
+      - ref: server.address
+        requirement_level: required
+        brief: >
+          Host identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+        note: >
+          If an HTTP client request is explicitly made to an IP address, e.g. `http://x.x.x.x:8080`, then
+          `server.address` SHOULD be the IP address `x.x.x.x`. A DNS lookup SHOULD NOT be used.
+      - ref: server.port
+        requirement_level:
+          conditionally_required: If not the default (`80` for `http` scheme, `443` for `https`).
+        brief: >
+          Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+      - ref: url.scheme
+        requirement_level: recommended
+        examples: ['http', 'https', 'ftp']
+
+  - id: metric.http.client.connection.duration
+    type: metric
+    metric_name: http.client.connection.duration
+    stability: experimental
+    brief: "The duration of the successfully established outbound HTTP connections."
+    instrument: histogram
+    unit: "s"
+    attributes:
+      - ref: network.peer.address
+        requirement_level: recommended
+      - ref: network.protocol.version
+        requirement_level: recommended
+        brief: HTTP protocol version of the connection in the connection pool.
+        note: >
+          HTTP 1.0 and 1.1 requests share connections in the connection pool and are both reported as version `1.1`.
+          So, the `network.protocol.version` value reported on connection metrics is different than the one reported
+          on request-level metrics or spans for HTTP 1.0 requests.
+        examples: ["1.1", "2", "3"]
+      - ref: server.address
+        requirement_level: required
+        brief: >
+          Host identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+        note: >
+          If an HTTP client request is explicitly made to an IP address, e.g. `http://x.x.x.x:8080`, then
+          `server.address` SHOULD be the IP address `x.x.x.x`. A DNS lookup SHOULD NOT be used.
+      - ref: server.port
+        requirement_level:
+          conditionally_required: If not the default (`80` for `http` scheme, `443` for `https`).
+        brief: >
+          Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+      - ref: url.scheme
```

Comment:
I think we need to add `error.type`, but this goes our of scope of this PR, I'll create an issue